services/QuillLMS/app/controllers/teachers/progress_reports/diagnostic_reports_controller.rb

```diff
@@ -79,7 +90,20 @@ def recommendations_for_classroom
   end
 
   def lesson_recommendations_for_classroom
-    render json: {lessonsRecommendations: get_recommended_lessons(current_user, params[:unit_id], params[:classroom_id], params[:activity_id])}
+    classroom = current_user.classrooms.find(params[:classroom_id])
```

Comment:
I think we want to use `classrooms_i_teach` here.

Reply:
Cissy is right, calling `.classrooms` on a user looks for `classroom_students` relationships, not `classrooms_teacher`.

Reply:
Nice catch, I'll make that change.